components/src/molecules/select/Select.stories.js

```diff
@@ -0,0 +1,98 @@
+import { storiesOf } from '@storybook/vue'
+import { action } from '@storybook/addon-actions'
+
+import countries from '../../../mocks/countries.json'
+
+import ASelect from './Select.vue'
+
+storiesOf('Molecules/Select', module)
+  .addParameters({ info: true })
+  .add(
+    'Default',
+    () => ({
+      components: { ASelect },
+      data () {
+        return {
+          countries,
+          selectedCountry: ''
+        }
+      },
+      methods: {
+        showCountry: action('Selected country')
+      },
+      template: `
+        <a-select
+          id="field-id"
+          name="field-name"
+          :options="countries"
+          v-model="selectedCountry"
+          @input="showCountry"
+        >
+          <template v-slot:label>
+            Label text
+          </template>
+          <template v-slot:option>
```

Comment:
please use shortcut from vue 2.6.0 in all places, then can go to develop

Reply:
Done